mne_bids/read.py

```diff
@@ -151,9 +151,18 @@ def _read_events(events, event_id, raw, bids_path=None):
                 'To specify custom event codes, please pass "event_id".'
             )
         else:
+            special_annots = {"BAD_ACQ_SKIP"}
             desc_without_id = sorted(
                 set(raw.annotations.description) - set(event_id.keys())
             )
+            # auto-add entries to `event_id` for "special" annotation values
+            # (but only if they're needed)
+            if set(desc_without_id) & special_annots:
+                for annot in special_annots:
+                    # use a value guaranteed to not be in use
+                    event_id = {annot: max(event_id.values()) + 1} | event_id
```

Comment:
Could we perhaps go much higher? Like, + 10000 or so? To clearly separate those events from the rest

Reply:
@sappelhoff If you feel this is unnecessary, go ahead and merge 

it was just a thought

Reply:
Did you mean this?

```suggestion
                    event_id = {annot: max(event_id.values()) + 10000} | event_id
```

I think it's a reasonable point 👍